bitcoin/Cargo.toml

```diff
@@ -34,6 +34,7 @@ hex = { package = "hex-conservative", version = "0.1.1", default-features = fals
 hex_lit = "0.1.1"
 internals = { package = "bitcoin-internals", version = "0.2.0" }
 io = { package = "bitcoin-io", version = "0.1", default-features = false, features = ["alloc"] }
+ordered = "0.1.1"
```

Comment:
Maybe we can make it optional in case people don't need it? It's a pretty uncommon case to need to order timelocks anyway.

Reply:
Yes, optional sounds better. Thanks